server/webapp/WEB-INF/rails/webpack/views/global/_variables.scss

```diff
@@ -104,3 +104,5 @@ $btn-text-color: #fff;
 $btn-reset: #666;
 $btn-danger: $go-danger;
 
+// tables
+$selected-table-row: #e2f7fa;
```

Comment:
where are other styles for the table?
How do we know what is the table row color? and upon selection this color will go well with the other unselected table rows?

Reply:
The other table styles are in the table component. However, we are not using the table component for users page. So added variable in default variables.